shenyu-client/shenyu-client-http/shenyu-client-springmvc/src/test/java/org/apache/shenyu/client/springmvc/init/SpringMvcClientEventListenerTest.java

```diff
@@ -183,4 +193,33 @@ public String test(final String hello) {
         }
     }
 
+    @RestController
+    @ShenyuRequestMapping("/order")
+    static class SpringMvcClientTestBean5 {
+
+        @ShenyuPostMapping(value = "/save")
+        public String save(@RequestBody final String body) {
+            return "" + body;
+        }
+
+        @ShenyuGetMapping("/{id}")
```

Comment:
You need to modify the `shenyu-admin` module to adapt the writing of `${id}`, use regular expression to replace `/aa/${xxx}/cc` to `/aa/**/cc`

Reply:
hi @dragon-zhang , Is that for rule matching?  i`ll modify and test it tomorrow.

Reply:
> hi @dragon-zhang , Is that for rule matching? i`ll modify and test it tomorrow.

yes